### PR TITLE
Release 2.9.12-beta3

### PR DESCRIPTION
**Pull request by tidy-dev**
Holding in draft until we get any remaining PR's merged. :) 

## Description
Looking for the PR for the upcoming third beta of the v2.9.12 series? Well, you've just found it, congratulations!

Things not user facing (thus, not reflected in release notes):
- Fix some \"checks failed\" notification metrics - #14214,
- Upgrade to Prettier 2.6.0 - #14205,
- Bump lodash from 4.17.11 to 4.17.21 - #14195,
- Fix permission issue when loading extension - #14183.,
- Replace issue templates with issue forms - #14126,
- The markdown parser correctly set's it's height relative to it's content - #14196
- Persist GUID to local file instead of local storage - #14192.

## Release checklist

- [x] Check to see if there are any errors in Sentry that have only occurred since the last production release
- [x] ~~Verify that all feature flags are flipped appropriately~~
- [x] ~~If there are any new metrics, ensure that central and desktop.github.com have been updated~~

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,10 @@
 {
   "releases": {
+    "2.9.12-beta3": [
+      "[Fixed] Escape Special Characters when adding a file to .gitignore - #10253. Thanks @uttiya10!",
+      "[Fixed] The markdown parser correctly set's it's height relative to it's content - #14196",
```
Comment (sergiou87) [inline changelog.json]:
```suggestion
      "[Fixed] Escape special characters when adding a file to .gitignore - #10253. Thanks @uttiya10!",
```

AFAIK #14196 is not user facing, right? We're not using `SandboxedMarkdown` anywhere in the app in prod or beta.

**Review by sergiou87 — CHANGES_REQUESTED** [review]

File: changelog.json (hunk 1 above)
Comment (tidy-dev) [reply changelog.json]: Yep - You are right. I guess I spend so much time in it I forgot. 😸

**Review by sergiou87 — DISMISSED** [review]

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,9 @@
 {
   "releases": {
+    "2.9.12-beta3": [
+      "[Fixed] Escape special characters when adding a file to .gitignore - #10253. Thanks @uttiya10!",
+      "[Fixed] The `Not Now` button in \"initialize LFS\" dialog dismisses the dialog - #14187"
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Dismiss \"initialize LFS\" dialog when clicking `Not Now` button - #14187"
```

**Review by billygriffin — CHANGES_REQUESTED** [review]
One minor change but otherwise looks good!

**Review by billygriffin — APPROVED** [review]
🚀

**Pull request merged**